styles/projectstyle.css

```diff
@@ -0,0 +1,81 @@
+* {
+    box-sizing: border-box;
+}
+
+html {
+    height: 100vh;
+}
+
```

Comment:
This code already exists in the main .css so you can remove it

Reply:
Problem fixed!